tool/lib/commands/update_flutter_sdk.dart

```diff
@@ -100,16 +90,28 @@ class UpdateFlutterSdkCommand extends Command {
       '${useCachedVersion ? 'from cache' : 'from upstream'}: $flutterTag ',
     );
 
-    if (updateFlutterFromPath) {
-      final sdk = FlutterSdk.current;
+    final sdk = FlutterSdk.current;
+    final flutterSdkDirName = 'flutter-sdk';
+    final toolSdkPath = path.join(repo.toolDirectoryPath, flutterSdkDirName);
+    final toolFlutterSdkDirectory = Directory(toolSdkPath);
```

Comment:
nit: add a getter `toolFlutterSdkPath` to the DevToolsRepo class

Reply:
Done!